.travis.yml

```diff
@@ -2,16 +2,16 @@ language: android
 sudo: required
 dist: trusty
 
+before_install:
```

Comment:
What's the motivation for this change?

Reply:
Travis sucks. 😆 Nah, just kidding. Seriously though, Travis has been unreliable in my experience when it comes to installing dependencies properly, hence trying to minimize the use of their scripts. For a concrete reason though, their `licenses` block [failed on one of my builds](https://travis-ci.org/SUPERCILEX/glide/jobs/309892702#L1204). If you can find another reliable solution that's less ugly than this one, awesome! Otherwise, I'm not sure what else to do. 😄

Reply:
Huh, I haven't seen that failure before. Maybe it's worse with pull requests? Right now my travis flakiness is all emulator related.

Reply:
I've also seen it with FirebaseUI: https://github.com/firebase/FirebaseUI-Android/pull/991#discussion_r146145121. It seems to happen when the Gradle plugin tries to install dependencies: I think Travis uses an `echo "y"` whereas Google actually looks for the files.

Reply:
Got it, I'm ok with it, thanks.